packages/sdk/src/base/items.ts

```diff
@@ -13,20 +13,22 @@ export class ItemsHandler<T extends Item> implements IItems<T> {
 		this.endpoint = collection.startsWith('directus_') ? `/${collection.substring(9)}` : `/items/${collection}`;
 	}
 
-	async readOne(id: ID, query?: QueryOne<T>): Promise<OneItem<T>> {
+	async readOne(id: ID, query?: QueryOne<T>, options?: TransportRequestOptions): Promise<OneItem<T>> {
 		const response = await this.transport.get<T>(`${this.endpoint}/${encodeURI(id as string)}`, {
+			...options,
 			params: query,
```

Comment:
```suggestion
			params: query,
			...options,
```

By doing this way, developers who want to override the default behaviour will be able to. The same applies to all modifications.

Reply:
done 👍🏻